jetty-cdi/src/main/java/org/eclipse/jetty/cdi/CdiServletContainerInitializer.java

```diff
@@ -36,9 +36,9 @@
  * the "jetty.cdi.mode" init parameter or default to the mode set by the
  * "jetty.cdi.mode" server attribute.  Supported modes are:</p>
```

Comment:
See comment above about the _property_ "jetty.cdi.mode" versus the attribute name "org.eclipse.jetty.cdi" - I think the javadoc should be fixed.